hands-on-exercise/5-ibc-adv/8-ibc-app-checkers.md

```diff
@@ -77,6 +77,59 @@ message Board {
 }
 ```
 
+Note that you will also have to modify the `x/leaderboard/genesis.go`. In it, look for:
+
+```golang
+// Set if defined
+    if genState.Board != nil {
+        k.SetBoard(ctx, *genState.Board)
+    }
+```
+
+and simply change it into:
+
+```golang
+    k.SetBoard(ctx, genState.Board)
+```
+
+And in the `x/leaderboard/genesis_test.go`, look for:
+
+```golang
+        Board: &types.Board{
+            PlayerInfo: new(types.PlayerInfo),
+        },
+```
+
+and instead use:
+
+```golang
+    Board: types.Board{
+     PlayerInfo: []types.PlayerInfo{},
+   },
+```
+
+We gave checker's module access to leaderboard's keeper. Therfore you will need to modify `testutils/keeper/checkers.go`, find:
+
+```golang
+k := keeper.NewKeeper(
+        bank,
+        cdc,
+        storeKey,
+        memStoreKey,
+```
+
+and add leaderboard's keeper into it:
+
+```golang
+    leaderboardKeeper,_ := LeaderboardKeeper(t);
+    k := keeper.NewKeeper(
+        bank,
+        *leaderboardKeeper,
+        cdc,
+        storeKey,
+        memStoreKey,
```

Comment:
I would adjust the indentation on all of them. 4 spaces, No `TAB` and always start at the same level a the backticks.